src/EdFi.AnalyticsMiddleTier.Tests/Common/InstallBaseTest.cs

```diff
@@ -33,18 +33,18 @@ abstract class When_running_installation
         protected const string QuickSightEWSDirectory = "Qews";
 
         protected IDatabaseMigrationStrategy MigrationStrategy { get; set; }
-        protected DatabaseUpgradeResult migrationResult = new DatabaseUpgradeResult(new List<SqlScript>(), true, null);
-        protected DatabaseUpgradeResult DatabaseAccessError = new DatabaseUpgradeResult(new List<SqlScript>(), false, new Exception(DatabaseAccessErrorMessage));
+        protected DatabaseUpgradeResult MigrationResult { get; set; }
+        protected DatabaseUpgradeResult DatabaseAccessError { get; set; }
```

Comment:
I might have been wrong about this change. It was flagged because the constructor for `DatabaseUpgradeResult` is [now obsolete](https://github.com/DbUp/DbUp/blob/e7d441b81e9c14af8f200e041ede6248aad8f71e/src/dbup-core/Engine/DatabaseUpgradeResult.cs#L16) and thus might break in the next upgrade of DbUp. Instead of directly instantiating an object, I wanted to see if we could get away with leaving as null initially. Furthermore, we prefer to use Properties rather than instance variables when the attribute will be exposed outside the class (that is, when not `private`). Let's see what happens in the unit testing.